sirepo/auth_db.py

```diff
@@ -79,10 +82,10 @@ def _add(proprietary_code_dir, sim_type, sim_data_class):
                     stderr=subprocess.STDOUT,
                 )
                 l = sirepo.simulation_db.simulation_lib_dir(sim_type)
-                assert not pykern.pkio.sorted_glob(l.join('*')), \
-                    f'expecting simulation_lib_dir={l} to be empty'
+                e = [f.basename for f in pykern.pkio.sorted_glob(l.join('*'))]
```

Comment:
I've never seen you use a frozenset for a local variable. Why would we use one here?

Reply:
frozenset speaks to the intention. It's a loop invariant. If Python optimizes anything, it can use that intention. More importantly, a reader can look at it and say that it is a loop invariant so it reduces cognitive load. I didn't know about frozenset until relatively recently.